src/main/java/com/esri/samples/localserver/local_server_dynamic_workspace_raster/README.md

```diff
@@ -0,0 +1,42 @@
+<h1>Local Server Dynamic Workspace Raster</h1>
+
+<p>Demonstrates how to dynamically add a local raster file to map using the Local Server. </p>
+
+<p><b>Note:</b> Local Server is not supported on MacOS</p>
+
+<img src="LocalServerDynamicWorkspaceRaster.png"/>
```

Comment:
The image needs to be exactly 800w x 400h. I usually create a new file in Gimp with the correct size. Then I paste my screenshot into the file. Gimp will keep the boundary so it gets cropped to the correct size.